packages/webapp-alternative/src/components/FunctionalComponent.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react'
+
+const FunctionalComponent = () => {
+  const [count, setCount] = useState(0);
+
+  //increase counter
```

Comment:
Let's remove redundant comments.

Reply:
Ahhaha, meaningless self-evident comments, mea culpa 😄  I'll remove it ✂️

Reply:
Done